### PR TITLE
CDK: Improve `filter_secrets` skip empty secret

### PR DESCRIPTION
**Pull request by grubberr**
Signed-off-by: Sergey Chvalyuk <grubberr@gmail.com>

## What
`source-bing-ads` has empty secret value on purpose:
```
$ cat secrets/config.json |  jq .client_secret
""
```

it produce "strange" log output like this:
```
{"type": "LOG", "log": {"level": "INFO", "message": "****C****h****e****c****k**** ****s****u****c****c****e****e****d****e****d****"}}
```

## How
Don't replace empty string with 4 starts

### CODE REVIEW

File: airbyte-cdk/python/airbyte_cdk/utils/airbyte_secrets_utils.py [hunk 1]
```diff
@@ -67,5 +67,6 @@ def filter_secrets(string: str) -> str:
     # TODO this should perform a maximal match for each secret. if "x" and "xk" are both secret values, and this method is called twice on
     #  the input "xk", then depending on call order it might only obfuscate "*k". This is a bug.
     for secret in __SECRETS_FROM_CONFIG:
-        string = string.replace(str(secret), "****")
+        if secret:
```
Comment (sherifnada) [inline airbyte-cdk/python/airbyte_cdk/utils/airbyte_secrets_utils.py:70]: could you unit test this behavior please?

Comment (grubberr) [reply airbyte-cdk/python/airbyte_cdk/utils/airbyte_secrets_utils.py:70]: ok I will add now

**Review by arsenlosenko — APPROVED** [review]

**Review by bazarnov — APPROVED** [review]

**Review by sherifnada — APPROVED** [review]

**Comment by edgao** [comment]
what's the process for getting this update into source-bing-ads? Do I just need to publish a new version?

**Comment by grubberr** [comment]
```
user@hrk1-ldl-A02942:~$ docker run -ti --rm --entrypoint /bin/bash airbyte/source-bing-ads:0.1.10 
root@c15119154049:/airbyte/integration_code# pip3 freeze | grep -i cdk
airbyte-cdk==0.1.74
```

@edgao yes we just need to re-release `source-bing-ads` to catch up new version of `airbye_cdk==0.1.75`

**Pull request merged**